test/xcode-specs.js

```diff
@@ -70,7 +81,7 @@ describe('xcode @skip-linux', () => {
     let path = await xcode.getAutomationTraceTemplatePath();
 
     should.exist(path);
-    fileExists(path).should.eventually.be.true;
+    fs.exists(path).should.eventually.be.true;
```

Comment:
Actually you still need:
`await  exists(path).should.eventually.be.true`
Otherwise it does not wait for it.
(The result is a promise.)